metricbeat/module/windows/service/defs_service_windows.go

```diff
@@ -51,6 +51,21 @@ var serviceErrors = map[ServiceErrno]struct{}{
 	SERVICE_ERROR_SERVICE_DOES_NOT_EXIST:  struct{}{},
 }
 
+type ServiceConfigInformation uint32
+
+const (
+	ServiceConfigDelayedAutoStartInfo   ServiceConfigInformation = 3
```

Comment:
I actually tried this on 3 different machines. I always get 

![image](https://user-images.githubusercontent.com/19551763/32980264-6f96f0fe-cc63-11e7-976b-11696cbd7a5e.png)

Reply:
That must be because those are `#define`s in the header file. In that case I would move these declarations out of this file because it's primary purpose is to specify constants by name and have them converted to their values.